src/components/Registration/RegistrationStepOne/RegistrationStepOne.tsx

```diff
@@ -168,32 +213,76 @@ export const RegistrationStepOne = withRouter(
-                            </Label>
-                          </Col>
-                          <Col sm="8">{pecRadioButtons}</Col>
-                        </FormGroup>
-                        <FormGroup row={true} className="mb-5">
-                          <Col sm="4">
-                            <Label htmlFor="text-input">
-                              {t("signUp.stepOne.inputs.scopeLabel")}
-                            </Label>
-                          </Col>
-                          <Col sm="8">{scopeRadioButtons}</Col>
-                        </FormGroup>
+                        {props.selectedAdministration.name ? (
+                          <Fragment>
+                            <FormGroup row={true}>
+                              <Col sm="4">
+                                <Label htmlFor="text-input">
+                                  {t("signUp.stepOne.inputs.pecLabel")}
+                                </Label>
+                              </Col>
+                              <Col sm="8">
+                                <RegistrationStepOneRadioButtons
+                                  name="selectedPecLabel"
+                                  disabled={
+                                    props.isAdministrationAlreadyRegistered
+                                  }
+                                  dataArray={pecsArray}
+                                  defaultCheckedValue={
+                                    props.selectedAdministration
+                                      .selected_pec_label
+                                  }
+                                  errors={errors}
+                                  errorText={requiredInputErrorText}
+                                  register={register}
+                                  setValue={setValue}
+                                  unregister={unregister}
+                                />
+                              </Col>
+                            </FormGroup>
+                            <FormGroup row={true} className="mb-5">
+                              <Col sm="4">
+                                <Label htmlFor="text-input">
+                                  {t("signUp.stepOne.inputs.scopeLabel")}
+                                </Label>
+                              </Col>
+                              <Col sm="8">
+                                <RegistrationStepOneRadioButtons
+                                  name="organizationScope"
+                                  disabled={
+                                    props.isAdministrationAlreadyRegistered
+                                  }
+                                  dataArray={organizationScopes}
+                                  defaultCheckedValue={
+                                    props.selectedAdministration.scope
+                                  }
+                                  errors={errors}
```

Comment:
why do we need to pass these value here ? can't we just use `useForm` inside `RegistrationStepOneRadioButtons `?

Reply:
if you mean declare `useForm` inside `RegistrationStepOneRadioButtons` , I just tried and it does not work (useForm declaration must be single, if you declare multple times checks are not performed)
this structure comes from React Form Hook documentation about using it with dinamic components: https://react-hook-form.com/advanced-usage#ConditionalControlledComponent